development/kops/install.sh

```diff
@@ -0,0 +1,8 @@
+set -eo pipefail
+curl -LO https://github.com/kubernetes/kops/releases/download/v1.18.2/kops-linux-amd64
```

Comment:
I *think* we don't need this curl and this file has no copyright. I think you meant to delete this file.

Reply:
Oh yes I wanted to delete this file, I'll do that in my next revision